composer.json

```diff
@@ -33,12 +33,8 @@
     "replace": {
         "typo3/eel": "self.version",
         "typo3/flow": "self.version",
-        "symfony/polyfill-php54": "*",
-        "symfony/polyfill-php55": "*",
-        "symfony/polyfill-php56": "*",
         "symfony/polyfill-php70": "*",
         "symfony/polyfill-php71": "*",
-        "symfony/polyfill-php72": "*",
         "symfony/polyfill-mbstring": "*",
```

Comment:
this one does not require `ext-mbstring` either

Reply:
It does in line 31…

Reply:
indeed. I missed it as it is not sorted with other extensions